deps.edn

```diff
@@ -1,12 +1,16 @@
-{:paths ["src"]
+{:paths ["src" "resources"]
```

Comment:
Huh, I think we want to add `resources` to the `pom.xml` too. I'm guessing it was automatically added to the jar when we used Leiningen, and perhaps it was removed when we moved to Clojure CLI.

```
    <resources>
      <resource>
        <directory>resources</directory>
      </resource>
    </resources>
```

Reply:
Done, thanks \o